src/vsphere_cpi/lib/cloud/vsphere/resources/network.rb

```diff
@@ -0,0 +1,110 @@
+require 'cloud/vsphere/resources'
+
+module VSphereCloud
+  module Resources
+    class Network
+      include VimSdk
+      include ObjectStringifier
+      stringify_with :name, :mob, :bosh_name
+
+      attr_reader :mob, :client, :bosh_name
+      def self.make_network_resource(bosh_name, mob, client)
```

Comment:
I don't think we need `bosh_name` at all if we're not populating `dvs_index`. Instead of `#get_dvs_index_hash`, can we make a function like `#index_name` that returns the index key that should go in such a hash? That way this network doesn't need to know about the BOSH name whatsoever.

Reply:
for now, yes we can make that change. 

But being a network resource object of CPI  I think it is fine to put the variable in that class too

Reply:
Actually maybe `#network_id` is a better name?

Reply:
Done